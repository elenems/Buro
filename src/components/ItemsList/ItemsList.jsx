import React from "react";
import ListItem from "./ListItem";
import Box from "@material-ui/core/Box";

export default function ItemsList(props) {
  return (
    <Box display="flex" justifyContent={props.justifyContent} flexWrap="wrap">
      {props.items.map(item => {
        return <ListItem key={item.id} item={item} />;
      })}
    </Box>
  );
}
