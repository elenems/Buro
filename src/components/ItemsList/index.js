import ItemsList from './ItemsList.jsx';
export default ItemsList;