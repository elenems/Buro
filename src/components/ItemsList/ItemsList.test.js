import React from "react";
import { shallow } from "enzyme";
import ItemsList from "../ItemsList";


describe("ItemsList", () => {

    it("Matches snapshot", () => {
    const wrapper = shallow(<ItemsList />).debug();
    expect(wrapper).toMatchSnapshot();
  });

  it("Renders at least one ListItem", () => {
    const wrapper = shallow(<ItemsList />);
    wrapper.setState({ items: [{ id: 1, title: "Title" }] });
    expect(wrapper.find("ListItem").length).toBeGreaterThan(0);
  });

  it("Ss", () => {
    const wrapper = shallow(<ItemsList />);
    wrapper.instance().componentDidMount();
  });
});
