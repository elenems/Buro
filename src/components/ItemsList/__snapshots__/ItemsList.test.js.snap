// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemsList Matches snapshot 1`] = `"<Styled(MuiBox) display=\\"flex\\" justifyContent={[undefined]} flexWrap=\\"wrap\\" />"`;
